packages/polaris-viz/src/components/Labels/SingleTextLine.tsx

```diff
@@ -48,14 +54,15 @@ export function SingleTextLine({
         height={LINE_HEIGHT}
         width={targetWidth}
         fill={color}
-        fontSize={FONT_SIZE}
+        fontSize={fontSize}
+        fontWeight={fontWeight}
         fontFamily={FONT_FAMILY}
         y={y}
         x={x}
       >
         {truncated}
       </text>
-      <title>{text}</title>
+      {willTruncate && <title>{text}</title>}
```

Comment:
Before this change it would always display the title so we default to `true`. There should be no change aside from my new case that I pass `false` to `willTruncate`